### PR TITLE
Add Prim to homepage

### PR DESCRIPTION
**Pull request by joneshf**
N.B. I haven't tried to run this. But I wanted to throw this out for discussion/opinion.

Should address #291.

Currently, there's no link at all to this page on Pursuit, any other PureScript website, or any other PureScript GitHub repo. What that means is that unless you're in the know, you can't find the page. I get that search results would be really great to have, but that shouldn't prevent anyone in the community from finding these docs.

I'm not saying this is the greatest thing ever, and I know it can be improved. But, this gets us from zero to nonzero, which is more valuable than being stuck trying to go from zero to ten.

### CODE REVIEW

**Comment by hdgarrood** [comment]
> Currently, there's no link at all to this page on Pursuit, any other PureScript website, or any other PureScript GitHub repo

This isn't actually true: whenever any `Prim` type appears in a type signature, it is rendered as a link to this module. Try clicking on any occurrence of `Int`, `String`, `Array` etc in <https://pursuit.purescript.org/packages/purescript-prelude/3.1.0/docs/Prelude>, for instance.

Anyway I think it would make the most sense to put this link in the same sentence as the one that suggests looking at the Prelude, which appears slightly earlier in the HTML.

**Comment by joneshf** [comment]
Moved to the same sentence.

**Comment by hdgarrood** [comment]
Looks good, thanks! This is definitely an improvement, although I think we could take this further still by explaining what Prim contains, so that people who don’t already know what Prim is are more likely to be able to benefit from it. At that point I think it’s likely to get a bit too wordy for the front page though, so I’d suggest we put together a separate page for this. This page could give you an overview and suggested perusal order for the core libraries, too: see #326.

Anyway I’m going to go ahead and merge this now, we can do all that later.

**Comment by joneshf** [comment]
Yeah, makes sense. Thanks!

**Comment by joneshf** [comment]
What do we need to do to update https://pursuit.purescript.org/ to show this?

**Comment by hdgarrood** [comment]
I can’t remember the exact details why this is the case but we need to wait until 0.12 of the compiler is out.

**Comment by hdgarrood** [comment]
(We could also backport but I’ve generally not bothered with backporting things because it takes up time that I don’t really have nowadays.)

**Comment by joneshf** [comment]
@hdgarrood Did 0.11.7 fix whatever the problem was? If so, can we update the site?

**Comment by hdgarrood** [comment]
The issue was that lots of links for types rendered in documentation were broken. I think it should be fixed in 0.11.7; when I get a moment, I'll try updating Pursuit to use 0.11.7 and check. Thanks for the reminder :)
